docs/guide.md

```diff
@@ -73,6 +73,17 @@ See also [Setting the permission policy](http://www.django-rest-framework.org/ap
     permission_classes = [HasAPIKey | IsAuthenticated]
     ```
 
+### Manually validating API keys
+You can also manually validate an API key with the `APIKey` objects manager. This is useful for validating API keys outside of a normal Django view, such as inside a websocket consumer from Django Channels, etc. Just use the `is_valid()` method on the manager like below.
```

Comment:
Tweaking the language a bit here. When writing docs, it's best not to use "just" or "simpli"y as things that seem obvious to us docs writers actually aren't to most readers (which is why they're documented! :) ).

```suggestion
You can also manually validate an API key with the `APIKey` objects manager using the `is_valid()` method on the manager lke below. This is useful for validating API keys outside of a normal Django view, such as inside a websocket consumer from Django Channels.
```